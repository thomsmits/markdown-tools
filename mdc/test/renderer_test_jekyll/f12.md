# Chapter
## Slide Footnote

Text using a footnote[^1] and another one[^label].

[^1]: Footnote with number.
[^label]: Footnote with label.
