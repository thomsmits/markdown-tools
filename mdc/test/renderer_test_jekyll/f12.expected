---
layout: page
parent: "Section 3"
title: "Chapter"
nav_order: 0
---
<h1 class="fw-700 text-purple-300">Chapter</h1>
## Slide Footnote

<p>Text using a footnote<a href="#footnote_1"><sup>1</sup></a> and another one<a href="#footnote_label"><sup>label</sup></a>.</p>

<p class="fs-2 text-blue-100" id="footnote_1">1. Footnote with number.</p>
<p class="fs-2 text-blue-100" id="footnote_label">label. Footnote with label.</p>
